Makefile

```diff
@@ -1,4 +1,4 @@
-RELEASE_DATE := "30-Nov-2017"
+RELEASE_DATE := "30 November 2017"
```

Comment:
I would argue just remove release date.  it doesn't give a lot of value to anything these days.  It's just one more thing to forget to do when tagging a new release.

Reply:
The Release date will be changed when the new version released. 
So the value should be limited. but  it is ok for me to change here.